config/database.yml

```diff
@@ -1,6 +1,8 @@
 default: &default
   adapter: postgresql
-  host: localhost
+  host: <%= ENV['GLOWFIC_DATABASE_HOST'] || 'localhost' %>
+  username: <%= ENV['GLOWFIC_DATABASE_USER'] || 'postgres' %>
```

Comment:
I _think_ it would work better to default this to `nil` if the ENV isn't set (ie remove the or condition), which will hopefully make it act like the line isn't set. My understanding is that, by default, Rails uses `throne3d` as the database user on my developer box, because I'm logged into a UNIX account named `throne3d` (and then I have to create a database user named `throne3d` and so on).

Reply:
Done =)